examples/question_answering_filtering_similar_answers.py

```diff
@@ -0,0 +1,24 @@
+from farm.infer import QAInferencer
```

Comment:
This might pollute our example section a bit much for such a small feature.

We can discuss if we want to introduce example folders based on tasks to organize the examples better.